product_docs/docs/migration_portal/4/known_issues_notes.mdx

```diff
@@ -37,6 +37,9 @@ The Migration Portal now supports assessing and migrating the Oracle users for t
 
 Since the grants are currently attached to the user and role objects, if one or more objects that the user is being granted a permission on fails, the user object itself fails assessment.  To resolve this issue, any grant statements on failed objects can be commented out either using a SQL single line comment (`--`) for individual statements or a SQL multi-line comment (`/* */`) for consecutive grant statements and then a reassessment of the object can be performed.  After the failed object corresponding to the failed grant statements have been repaired, the previously failing grant statement can be uncommented and the user object reassessed.
 
+!!!note
```

Comment:
Is there a reason why this was created as "Note" in this section?  The whole page is about known issues, limitations, and notes.  I think the note can be simply added as another paragraph in under the Grants section.